apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -992,8 +1002,8 @@ private void HandleStartStreamingPullWithoutBackoff()
                 {
                     SubscriptionAsSubscriptionName = _subscriptionName,
                     StreamAckDeadlineSeconds = _modifyDeadlineSeconds,
-                    MaxOutstandingMessages = _flow.MaxOutstandingElementCount,
-                    MaxOutstandingBytes = _flow.MaxOutstandingByteCount
+                    MaxOutstandingMessages = _useLegacyFlowControl ? 0 : _flow.MaxOutstandingElementCount,
```

Comment:
Are you sure this is the right way round? Given the description of the PR, I'd expected UseLegacyFlowControl=true to be the "do what we did before" setting, whereas this change makes the *default* be "as it is now".